src/views/ApartmentApp/apartmentApp.scss

```diff
@@ -1,84 +1,87 @@
 @import '../../vars';
 
-strong.over-emphasized {
-  text-transform: uppercase;
-  text-decoration: underline;
-}
+.student-apartment-application,
+.apartment-application {
+  & strong.over-emphasized {
+    text-transform: uppercase;
+    text-decoration: underline;
+  }
 
-.apartment-card-header {
-  background-color: $primary-blue;
-  color: $neutral-white;
-  padding: '10px';
-  .MuiTypography-colorTextSecondary {
-    color: $neutral-light-gray;
-    font-size: 1.25rem;
+  & .apartment-card-header {
+    background-color: $primary-blue;
+    color: $neutral-white;
+    padding: '10px';
```

Comment:
Ah, that is probably an old copy-and-paste mistake,

It also makes me wonder why I ever put that there in the first place, since clearly I never even noticed that it isn't doing anything during recent testing, so why did past-me think we needed to add that padding...

Reply:
```suggestion
```
If the page looks fine the way it is now, then I would suggest just removing that padding because I don't see why that padding would need to be added